mne/viz/_brain/_brain.py

```diff
@@ -2482,10 +2479,10 @@ def add_text(self, x, y, text, name=None, color=None, opacity=1.0,
             background color).
         opacity : float
             Opacity of the text (default 1.0).
-        row : int
-            Row index of which brain to use.
-        col : int
-            Column index of which brain to use.
+        row : int | None
+            Row index of which brain to use. Default all rows.
+        col : int | None
+            Column index of which brain to use. Default all columns.
```

Comment:
It's not uncommon to want *text* in only one window (and bottom right is at least a somewhat reasonable default), and this is a backward incompatible change. I think I'd just leave the old default actually, we could deprecate but again I'm not 100% convinced text should be treated the same way as data.

Reply:
Ok I'll just leave the default as `-1` and document the bottom left behavior